src/www/ui/api/documentation/openapi.yaml

```
@@ -1144,7 +1144,32 @@ paths:
                 $ref: '#/components/schemas/Info'
         default:
             $ref: '#/components/responses/defaultResponse'
-
+  /groups/members/{id}:
```

Would rather recommend following format as followed by other endpoints as well.

```suggestion
  /groups/{id}/members:
```

If I were to translate the suggested URL, it can be done logically as:
Get me the property "members" of an individual "group" with an identifier "{id}"